docs/epics-and-tasks.md

```
@@ -0,0 +1,187 @@
+
+Backlog
+
+In Progress
+
+In Review
+
+Done
+
+Issues naming:
+
+[EPIC-1][TASK-1.2] OpenDota JSON parsing
```

markdownlint-cli2 (0.21.0)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: "task-1.2"

(MD052, reference-links-images)